src/components/usertask-component/form-fields/input_validator.ts

```diff
@@ -0,0 +1,17 @@
+export class InputValidator {
+  private readonly regex: RegExp;
```

Comment:
Private variables always get prefixed with an `_`.
```suggestion
  private readonly _regex: RegExp;
```